source/Halibut/Transport/SecureClient.cs

```diff
@@ -80,7 +80,6 @@ public void ExecuteTransaction(Action<MessageExchangeProtocol> protocolHandler)
                     log.Write(EventType.Error, $"The remote host at {(ServiceEndpoint == null ? "(Null EndPoint)" : ServiceEndpoint.BaseUri.ToString())} refused the connection, this may mean that the expected listening service is not running.");
                     lastError = cex;
                     retryAllowed = false;
-                    Thread.Sleep(retryInterval);
```

Comment:
Should have removed this from the other change. No need to sleep here, the outer loop will do that